### ISSUE TITLE
Enum case conversion

### ISSUE THREAD

**Issue by Dimentar** [ServiceStack/ServiceStack.Swift#9]
Hi, I have a request to improve Swift DTO code generation for enums. At this point .NET enums being UpperCase are converted also in UpperCase.
From swift [Swift Proposal 006](https://github.com/apple/swift-evolution/blob/master/proposals/0006-apply-api-guidelines-to-the-standard-library.md) enum cases are transformed to LowerCase.
It is possible to specify params for output: camelCase or lowercase? 
For Example: 

```
// C# input
enum EnumCamelCase {
    case First
    case Second
    ...
    case SomeCoolCase
    case OtherNotCoolCase
}

 // Swift output
enum EnumCamelCase {
    case first
    case second
    ...
    case someCoolCase
    case otherNotCoolCase
}
```

```
// C# input
enum EnumLowerCase {
    case ONE
    case TWO
}

 // Swift output
enum EnumLowerCase {
    case one
    case two
}
```

Thanks

**Comment by mythz** [comment]
I've added opt-in support for Swift-style enums from [this commit](https://github.com/ServiceStack/ServiceStack/commit/0b215e02d3a5bfb25bcaac4db969867cef142f4e) where you can choose to emit camel case enums with:

```csharp
SwiftGenerator.EnumNameStrategy = SwiftGenerator.SwiftStyleEnums;
```

`EnumNameStrategy` is just a `Func<string,string>` so you could also easily change it to use a custom strategy as well.

This change is available from v5.1.1 that's now [available on MyGet](http://docs.servicestack.net/myget).

**Comment by Dimentar** [comment]
Thanks a lot @mythz
